apps/dg/utilities/touch_tooltips.js

```diff
@@ -41,8 +41,9 @@ DG.TouchTooltips = (function() {
                     ? triggerView.get('layer')
                     : triggerView,
           container = DG.mainPage.getPath('mainPane.layer'),
-          _placement = placement || 'bottom-start',
+          _placement = placement || 'top-start',
```

Comment:
Changing the default placement to above the target element so the tooltip doesn't appear underneath the finger.